cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -183,6 +186,26 @@ var CMS = window.CMS || {};
                     );
                 });
 
+                // prevents single click from messing up the edit call
+                this.ui.container.find('a').on('click', function (e) {
+                    if (e.shiftKey || e.ctrlKey || e.metaKey) {
+                        return;
+                    }
+
+                    e.preventDefault();
```

Comment:
@vxsx added the check along with the minor comment, custom js handlers are going to work now.

another thing - I played around with the default addons (ex.accordions), and their handlers often aren't attached to `a`, but rely upon event propagation. So I suggest getting rid of `e.stopPropagation()` here.